services/common-features.yaml

```diff
@@ -1,13 +1,13 @@
 features:
-  - id: CCC.F01 # Encryption in Transit Enabled by Default
-    title: Encryption in Transit Enabled by Default
```

Comment:
enc in transit isn't a given or enables by default. But you can configure and use enc

Reply:
I was able to easily find documentation describing the places where GCP provides default encryption in transit as a key feature. If other providers aren't doing something similar already, I'd say they need to be held to the same standard.

https://cloud.google.com/docs/security/encryption-in-transit#encryption_in_transit_by_default

Reply:
If it is not enabled by default for the service that you are working on then I suggest don't include this common feature and create a specific feature for that service